### PR TITLE
Added TLS on YottaDB section

### PR DESCRIPTION
**Pull request by ranjani789**

### CODE REVIEW

**Comment by shabiel** [comment]
Thank you for taking my work in!

**Comment by ranjani789** [comment]
Thanks, Sam, for the clear explanation and layout! Made us realize it needed to be added to the documentation.

File: ProgGuide/ioproc.rst [hunk 1]
```diff
@@ -1812,6 +1812,71 @@ The ZSHOW "D" command reports available information on both the local and remote
...
+     7064420FDCAEE313B222 
+
+* In your environment file, set gtmtls_passwd_{section name} to be that hash.
+
+  .. parsed-literal::
+     export gtmtls_passwd_dev="7064420FDCAEE313B222" 
+
+* Start the server
+
+At this point, if you go to https://localhost:9081, you should be able to see the web pages via TLS.
```
Comment (shabiel) [inline ProgGuide/ioproc.rst]: I don't think this part belongs here.

**Comment by shabiel** [comment]
Looks good to me now.

**Pull request merged**